content/cpp/concepts/math-functions/terms/atan2/atan2.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'atan2()'
+Description: 'Returns the inverse tangent of coordinates in radians.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`atan2()`** function returns the inverse tangent of a coordinate (y/x), where y is the y-coordinate value and x is the x-coordinate value.
+
+## Syntax
+
+```pseudo
+atan2(y, x)
+```
+
+The `y`, `x` parameter can be one of `double`, `float` or `long double` datatype.
```

Comment:
```suggestion
The first and second parameters, `y` and `x`, can be one of `double`, `float` or `long double` [data type](https://www.codecademy.com/resources/docs/cpp/data-types).
```

Reply:
Can we also include 1-3 sentences about how the `atan2()` function behaves when either parameter is negative, or if both parameters are equal to zero (0)?